src/ui/components/URLBar/URLBar.tsx

```diff
@@ -128,7 +130,7 @@ export function URLBar() {
             text = null;
           } else {
             if (children.length) {
-              text = children;
+              text = children[0];
```

Comment:
looks like loss of data, can you comment this pls?

Reply:
Yes, sure!
This case is when the children is just a string. Not a component
Will write a comment